cl/search/models.py

```diff
@@ -1017,6 +1031,20 @@ class Meta:
     def __str__(self) -> str:
         return f"{self.pk} ---> {trunc(self.description, 50, ellipsis='...')}"
 
+    @property
+    def datetime_filed(self) -> datetime | None:
+        if self.time_filed:
+            from cl.recap.constants import COURT_TIMEZONES
+
+            local_timezone = pytz.timezone(
+                COURT_TIMEZONES.get(self.docket.court.id, "US/Eastern")
+            )
+            return make_aware(
+                datetime.combine(self.date_filed, self.time_filed),
+                local_timezone,
+            )
```

Comment:
Got it! yeah Django helper functions were not adding additional value and they were only hiding what's really going in here.

Changed, just instead of using  `return datetime.combine(self.date_filed, self.time_filed, tzinfo=local_timezone)`

I changed it to:
`return local_timezone.localize(datetime.combine(self.date_filed, self.time_filed))`

Since we're using a `pytz` timezone it has the `localize` property that takes care of DST offsets if the timezone observes DST.